Realm/Tests/NotificationTests.m

```diff
@@ -1727,3 +1727,362 @@ - (void)testModifyUnobservedKeyPathArrayProperty {
 }
 
 @end
+
+@interface MixedCollectionChangesetTests : RLMTestCase <ChangesetTestCase>
+@end
+
+@implementation MixedCollectionChangesetTests  {
+    MixedObject *_obj;
+}
+- (void)setUp {
+    RLMRealm *realm = [RLMRealm defaultRealm];
```

Comment:
This logic should be in `prepare` so that each of the `ExpectChange` calls has a clean state.